scripts/modify-schema.js

```diff
@@ -0,0 +1,42 @@
+var fs = require('fs')
```

Comment:
javascript? 😞

Reply:
Because it's easy to play with big JSON ;)

Reply:
It just sucks if you have to run it :-(

Reply:
thats why I put Docker image for it